sql/core/benchmarks/FilterPushdownBenchmark-jdk11-results.txt

```diff
@@ -2,669 +2,669 @@
 
-OpenJDK 64-Bit Server VM 11.0.10+9-LTS on Linux 5.4.0-1043-azure
-Intel(R) Xeon(R) Platinum 8171M CPU @ 2.60GHz
+OpenJDK 64-Bit Server VM 11.0.11+9-LTS on Linux 5.4.0-1046-azure
+Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
 Select 1 string row (value = '7864320'):  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Parquet Vectorized                                10632          10694          60          1.5         676.0       1.0X
-Parquet Vectorized (Pushdown)                       608            635          22         25.9          38.6      17.5X
-Native ORC Vectorized                              8790           8838          37          1.8         558.9       1.2X
-Native ORC Vectorized (Pushdown)                    559            584          22         28.1          35.5      19.0X
+Parquet Vectorized                                10228          10471         167          1.5         650.3       1.0X
+Parquet Vectorized (Pushdown)                       511            519           5         30.8          32.5      20.0X
+Native ORC Vectorized                              6700           6865         119          2.3         426.0       1.5X
+Native ORC Vectorized (Pushdown)                    436            454          12         36.1          27.7      23.5X
 
-OpenJDK 64-Bit Server VM 11.0.10+9-LTS on Linux 5.4.0-1043-azure
-Intel(R) Xeon(R) Platinum 8171M CPU @ 2.60GHz
+OpenJDK 64-Bit Server VM 11.0.11+9-LTS on Linux 5.4.0-1046-azure
+Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
 Select 1 string row (value <=> '7864320'):  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 -------------------------------------------------------------------------------------------------------------------------
-Parquet Vectorized                                 10529          10624          74          1.5         669.4       1.0X
-Parquet Vectorized (Pushdown)                        613            631          16         25.7          39.0      17.2X
-Native ORC Vectorized                               8746           8816          63          1.8         556.1       1.2X
-Native ORC Vectorized (Pushdown)                     589            600          11         26.7          37.5      17.9X
+Parquet Vectorized                                 10287          10449         144          1.5         654.0       1.0X
+Parquet Vectorized (Pushdown)                        467            494          20         33.7          29.7      22.0X
+Native ORC Vectorized                               6781           6848          58          2.3         431.1       1.5X
+Native ORC Vectorized (Pushdown)                     428            440          10         36.8          27.2      24.1X
```

Comment:
ditto. `17 vs 17` -> `22 vs 24`.

Reply:
No. Github action runs on different machines, there is a performance difference between them.